src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -114,29 +125,253 @@ Template:
+| Magnetization transfer ratio               | MTR     | Grouping | Anatomical images for calculating a semi-quantitative magnetization transfer ratio map. _Associated output suffixes_: MTRmap                                                                                                                                                                                                                                                                                                                                                                                 |
+| Magnetization transfer saturation          | MTS     | Grouping | Anatomical images for calculating a semi-quantitative magnetization transfer saturation index map. The MTS method involves three sets of anatomical images that differ in terms of application of a magnetization transfer RF pulse (MTon or MToff) and flip angle ([Helms et al. 2008](https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.21732)). _Associated output suffixes_: T1map, MTsat                                                                                                                                                                                               |
+| Multi-parametric mapping                   | MPM     | Grouping | Anatomical images for multiparametric mapping (a.k.a hMRI). The MPM approaches involves the acquisition of highly-similar anatomical images that differ in terms of application of a magnetization transfer RF pulse (MTon or MToff), flip angle and (optionally) echo time and magnitue/phase parts ([Weiskopf et al. 2013](https://www.frontiersin.org/articles/10.3389/fnins.2013.00095/full)). See [here](https://owncloud.gwdg.de/index.php/s/iv2TOQwGy4FGDDZ) for suggested MPM acquisition protocols.,_Associated output suffixes_:R1map, R2starmap, MTsat, PDmap, T1map, T2starmap                                               |
+
+For example:
+
+```Text
+sub-01_fa-1_VFA.nii.gz
+sub-01_fa-1_VFA.json
+sub-01_fa-2_VFA.nii.gz
+sub-01_fa-2_VFA.json
+```
+
+Please see the [entity table appendix]() for the REQUIRED and OPTIONAL entities
+for each `grouping suffix`. 
+
+Note that every image in a **grouped scan collection** has the same `_<suffix>` 
+as they are supposed to be used together. Although the acquisitions will have
+many identical acquisition parameters, only one-to-one-mapping is allowed
+between a `.json`-sidecar file and an image in a grouped scan collection.
+This follows directly from the
+[inheritance principles of BIDS](../02-common-principles.md#the-inheritance-principle):
+parameter values that are identical across a set of grouped scans will still
+have to be stored separately in each `.json`-sidecar file.
+
+##### qMRI map suffixes 
```

Comment:
Could/should this perhaps be moved to the Derivatives page? While qMRI maps can be created at the scanner, these are derived images and would primarily be created after acquisition, right?

Alternatively, it could be renamed to "Scanner-derived map suffixes", if the assumption is that these maps are generated at the scanner, since the same concept should apply to non-qMRI derived data.

Reply:
We had lenghty discussions on this. The probelm is that they can be both scanner-derived and pipline-derived.

Reply:
Recall also that any raw-like data can be turned into a derivative by adding `desc-` or `space-` keyword.

On the whole, I follow @yarikoptic's philosophy that the all data are derivative, and that blurring the line between raw and derivative isn't bad, as long as we preserve backwards compatibility.

Reply:
@tiborauer @effigies Both of your points make sense, but if we want to support non-qMRI derived suffixes in the base specification (given that scanners can derive data that is not quantitative), do we really want to create a _fourth_ suffix group? Wouldn't it be better to keep an eye toward future BEPs with the name for this group of suffixes?

Reply:
I agree with you, @tsalo. I can see two main kind of suffixes: 
- I would combine the existing Conventional suffixes and the qMRI map suffixes, because both of them refer to stand-alone acquitions. We could call them, perhaps, Stand-alone suffixes OSLT.
- The Grouping suffixes would refer to acquisitions, which belong together according to some specifed logic.

If desired, we can make subgroups by splitting the tables of, for example, Stand-alone suffixes into Contrast and Parameter (OSLT) suffixes.